sdk/nodejs/dagger/connect.ts

```diff
@@ -0,0 +1,147 @@
+// WIP(TomChv): This file shall be renamed to something else
+import { Client } from './client';
+import { execa, execaCommandSync } from 'execa';
+import path from 'path';
+import axios from 'axios';
+
+/**
+ * Cloak binary name
+ */
+const CLOAK_BINARY = "cloak";
+
+/**
+ * ConnectOpts defines option used to run cloak
+ * in dev mode.
+ * Options are based on `dagger cloak` CLI.
+ */
+export interface ConnectOpts {
+	LocalDirs?: Record<string, string>;
```

Comment:
According to https://github.com/dagger/dagger/pull/3265#discussion_r1010930750, shall we remove it in Ts SDK too?

Reply:
I would rather wait for the PR #3265 to be ready and merged. Then we can get rid of this part.

Reply:
Actually you're right, let's get rid of it.

Reply:
Done in https://github.com/dagger/dagger/pull/3652/commits/5d6905c876441a1cae0453ae5402b37aeeb83c24